pilot/pkg/xds/endpoints/endpoint_builder.go

```diff
@@ -335,6 +335,10 @@ func (b *EndpointBuilder) BuildClusterLoadAssignment(endpointIndex *model.Endpoi
 		if svcPort.Name != ep.ServicePortName {
 			return false
 		}
+		// filter out endpoint that has invalid ip address, mostly domain name. Because this is generated from ServiceEntry.
```

Comment:
Was this a regression?

Reply:
From the issue, it is

Reply:
start from 1.20.0, maybe 1.19

Reply:
It would be nice to confirm the regression so we know this is the correct fix

Reply:
+1. It would be good identify what commit caused this so that we can know what happened